plugins/techdocs/src/reader/transformers/addGitFeedbackLink.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Transformer } from './index';
+
+// requires repo
+export const addGitFeedbackLink = (): Transformer => {
+  return dom => {
+    const contentContainer = dom.querySelector(
+      '.md-content__inner',
+    ) as HTMLElement;
+    const sourceAnchor = dom.querySelector(
+      '.md-content__inner>a',
+    ) as HTMLAnchorElement;
+    // don't show if Gitlab or Github hosting not specified
+    if (!sourceAnchor || !sourceAnchor.href) {
+      return dom;
+    }
+    let gitHost = '';
+
+    if (sourceAnchor.href.includes('gitlab')) {
+      gitHost = 'gitlab';
+    } else if (sourceAnchor.href.includes('github')) {
+      gitHost = 'github';
+    } else {
+      return dom;
+    }
+
+    const title = (contentContainer.querySelector(
+      'h1',
+    ) as HTMLElement).textContent!.replace(/[^a-zA-Z ]/g, '');
```

Comment:
If possible, please add a comment here explaining what's happening.

Reply:
Updated

Reply:
Changed selector to .childNodes[0].textContent to avoid the regex, which was to strip the link character from title string